pkg/clients/signingclienttracker.go

```diff
@@ -115,6 +113,11 @@ func (ct *SigningClientTracker) Status() (s *statuspb.ProtocolStatus, err error)
 	return nil, nil
 }
 
+// The ImplementsModule method only serves the purpose of indicating that this is a Module and must not be called.
+func (ct *SigningClientTracker) ImplementsModule() {
+	panic("ImplementsModule must not be called")
```

Comment:
The `panic` thing feels a bit excessive to my taste, but if you think that it makes sense to make it explicit in the implementation, then ok.
A single comment in the interface definition could also probably get the job done :)

Reply:
Yes I was hesitating too, but in the end I kept it for completeness sake. The comment in the definition is now there too.

Reply:
I'd like to keep the comments. As a reader of that code I'm sure I'd appreciate them.
The panics probably are a bit over the top indeed, I'll remove them.